prepare.php

```diff
@@ -99,6 +98,11 @@
 	echo PHP_EOL;
 }
 EOT;
+$db_version = <<<EODB
+\$mysqli = new mysqli(DB_HOST, DB_USER, DB_PASSWORD, DB_NAME);
```

Comment:
this will be called outside of the hosting env, so it might cause issues, because not every DB might be publicly accessible.

Reply:
This is is escaped so its forming the code appended to `/wp-tests-config.php` which as far as I know is executed within the hosting environment.

Reply:
Ah, I see! but then we change the version from cli version to mysql version. I think this is even better, but does that lead to confusion?

Reply:
if its suddenly slightly different people should quickly realize the newer version is actually accurate.

Reply:
acceptable?